README.md

```diff
@@ -122,7 +122,7 @@ Run the index.html with your preferred browser.
 
```

Comment:
- [ ] I kindly suggest that you please add proper instructions for running your project locally like below:


## EXAMPLE

### Setup
```

Clone this repository to your desired folder:

````sh`
  cd my-folder
  git clone git@github.com:Bhekisile/Leaderboard.git
`

### Install

Install this project with:


- Install Node Modules

`   ```sh
`   npm install
`   ```
`
### Usage

To run the project, execute the following command:


- Start the development server

`   ```sh
`   npm run dev
`   ```
`
### Run tests

To run tests, run the following command:

- Test the items Counter and Comments Counter function

   ```sh
   npm test

```

Reply:
Hi,
I've fixed my README file accordingly but I left the run tests because there's no test. Please correct me if I'm wrong.
Thank you

Reply:
@Bhekisile, you're right. Feel free to leave out the tests